rllib/core/rl_module/tf/tf_rl_module.py

```diff
@@ -0,0 +1,62 @@
+from typing import Any, Mapping
```

Comment:
Can you create a new unittest for the TFRLModule itself? similar to the torch version on the new PR I shared?